todos/models.py

```diff
@@ -1,10 +1 @@
 from django.db import models
-
-class Todo(models.Model):
```

Comment:
Why have we removed the class Todo from the file. Please keep the file and add registration model.